lib/internal/streams/from.js

```diff
@@ -48,10 +49,13 @@ function from(Readable, iterable, opts) {
       const { value, done } = await iterator.next();
       if (done) {
         readable.push(null);
-      } else if (readable.push(await value)) {
-        next();
       } else {
-        reading = false;
+        if (value == null) throw new ERR_STREAM_NULL_VALUES();
+        if (readable.push(await value)) {
```

Comment:
I think it makes sense to first await `value`, and then check whether it is `null`? Otherwise this could still result in `readable.push(null)` if `value` is e.g. `Promise.resolve(null)`, right?

Reply:
+1! I would have done this in the initial commit itself had it occurred to me earlier. Thank you for spotting this, I've made the changes.